supplementary_style_guide/glossary_terms_conventions/general_conventions/f.adoc

```diff
@@ -82,6 +82,17 @@ Use the hyphenated "fault-tolerant" when using it as an adjective.
 
 *See also*:
 
+//Add FedRAMP terminology Issue #395
```

Comment:
Do we want to embed metadata like this? I don't see in in other cases.

Reply:
I agree with @rolfedh that maybe it's better not to add code comments that duplicate what's in the Git blame.

Reply:
@spank50 I think the only remaining change needed is to remove the comment in line 85. Can you please make this change and we'll try to get this PR approved? Many thanks :)

Reply:
I think the //Add FedRAMP... comment is removed.